docs/jax.rst

```diff
@@ -80,6 +80,7 @@ Just-in-time compilation (:code:`jit`)
     named_call
     named_scope
     block_until_ready
+    make_mesh
```

Comment:
While you are here, can you remove device_put_replicated and device_put_sharded please?

Reply:
Done!